src/background.js

```diff
@@ -4,6 +4,7 @@ import { app, protocol, BrowserWindow, ipcMain } from "electron";
 import { createProtocol } from "vue-cli-plugin-electron-builder/lib";
 import installExtension, { VUEJS3_DEVTOOLS } from "electron-devtools-installer";
 import FolderReader from "./services/folderReader";
+import constants from "../utils/constants";
```

Comment:
```suggestion
import { GET_FOLDER_PATH_EVENT, GET_FOLDER_PATH_REPLY } from "../utils/constants";
```
So you don't need to add `constants.CONST` everywhere